lib/WebService/Async/CustomerIO.pm

```diff
@@ -53,31 +53,23 @@ Parameters:
 
 =cut
 
-sub configure {
-    my ($self, %args) = @_;
+sub _init {
```

Comment:
I think `new` is more common

Reply:
After discussion, appreantly `_init` is used inside `new` in `IO::Async::Notifier`